nomos-cli/Cargo.toml

```diff
@@ -18,3 +18,4 @@ serde = { version = "1.0", features = ["derive"] }
 tokio = { version = "1", features = ["sync"] }
 tracing = "0.1"
 tracing-subscriber = "0.3"
+serde_json = "1.0.133"
```

Comment:
Lets use serde_json "1", this will pull latest version compatible with other workspace crates

Reply:
Fixed at https://github.com/logos-co/nomos-node/pull/994/commits/23f86dcd6e60887b23a85ce2bebf81e6625a7864